tests/Mockery/Matcher/PHPUnitConstraintTest.php

```diff
@@ -33,7 +34,18 @@ class PHPUnitConstraintTest extends \PHPUnit_Framework_TestCase
 
     public function setUp()
     {
-        $this->constraint = \Mockery::mock('PHPUnit_Framework_Constraint');
+        /*
+         * Revise by PHPUnit version
+         */
+        if (class_exists('\PHPUnit\Framework\AssertionFailedError')) {
```

Comment:
no need for a slash, php just deletes it
https://github.com/php/php-src/blob/d2285e1613f239043d439a6b439ee510452f5e07/Zend/zend_builtin_functions.c#L1376
https://github.com/php/php-src/blob/5340587c253fad8e6bbfbbf62372ba017970ea1a/Zend/zend_execute_API.c#L961